lady_luck.js

```diff
@@ -634,38 +590,52 @@ function wcShow(msg, user_id) {
 	msg.reply(user.toString() + " has " + wc[guild.id].users[user.id].bonus_points + " Bonus Points and is " + wc[guild.id].users[user.id].word_count + " words toward their next one.");
 }
 
-client.on('message', msg => {
-	words = msg.content.split(/\s+/);
-	command = words[0].toLowerCase();
-	if (command === '!nwod') {
-		var outcome = nwodToText(nwodRoll(words[1], words[2]));
-		console.log(`New nWoD dice roll from ${msg.author.username}#${msg.author.discriminator}. Outcome: ${outcome}`)
-		msg.reply(outcome);
-	} else if (command == "!init") {
-		nwodInitToText(msg, words[1], words[2]);
-	} else if (command === "!initforce") {
-		nwodInitForceToText(msg, words[1], words[2]);
-	} else if (command === "!initclear") {
-		nwodInitClear(msg);
-	} else if (command === "!wc-add") {
-		wcAdd(msg, words[1]);
-	} else if (command === "!wc-rem") {
-		wcRem(msg, words[1]);
-	} else if (command === "!wc-ooc") {
-		wcOOC(msg, words[1]);
-	} else if (command === "!wc-ra") {
-		wcRA(msg, words[1]);
-	} else if (command === "!wc-rr") {
-		wcRR(msg, words[1]);
-	} else if (command === "!wc-list") {
-		wcList(msg);
-	} else if (command === "!wc-force") {
-		wcForce(msg, words[1], words[2], words[3]);
-	} else if (command === '!wc') {
-		wcShow(msg, words[1]);
-	}
-
- 	wordCountConsider(msg);
+client.on('message', function (m) {
```

Comment:
This should be `function (msg)`. I just tried running your PR and it crashes on `msg` not being found.